docker-compose.yml

```diff
@@ -0,0 +1,30 @@
+# SPDX-License-Identifier: MIT
+# SPDX-FileCopyrightText: 2023 Ruchita Nathani <ruchita.nathani@fau.de>
+
+version: "3"
+services:
+  mongodb:
+    image: mongo:latest
+    ports:
+      - "27017:27017"
+    environment:
+      MONGO_INITDB_ROOT_USERNAME: root
+      MONGO_INITDB_ROOT_PASSWORD: example
+    networks:
+      - network_private
```

Comment:
Yes, but we don't have AWS credentials right now so for now I make it for local storage.
If you have any other approach then please let me know I can work for that.

Reply:
I was just wondering if we also need to specify a local directory to save the data persistently here.

Another note (perhaps also for future tasks) the secrets here can also be part of local .env files. This will become relevant when using a centralized DB with real sensitive data.

Reply:
For now, I pushed a local credential. However, I plan to store the credentials in a local env file for added security.